Makefile

```diff
@@ -49,7 +49,7 @@ unit-tests-race:
 	cd integration/nwo/; export FAB_BINS=$(FAB_BINS); go test -cover ./...
 
 .PHONY: docker-images
-docker-images: fabric-docker-images weaver-docker-images fpc-docker-images orion-server-images monitoring-docker-images
+docker-images: fabric-docker-images weaver-docker-images fpc-docker-images orion-server-images monitoring-docker-images testing-docker-images
```

Comment:
I guess the name of the command is temporary or is it for tests?

Reply:
It's for tests 😅. Do you have an idea for a better name?

Reply:
Oh ok I thought Postgres was going to be an option for production, my bad.